cirq/protocols/approximate_equality_protocol.py

```diff
@@ -94,6 +95,12 @@ def approx_eq(val: Any, other: Any, *, atol: Union[int, float] = 1e-8) -> bool:
     if isinstance(val, str):
         return val == other
 
+    if isinstance(val, sympy.Basic) or isinstance(other, sympy.Basic):
+        delta = sympy.Abs(other - val).simplify()
+        if not delta.is_number:
+            raise AttributeError('Cannot reduce Sympy expression to number.')
```

Comment:
one last nit: I would make the error message more descriptive to include contextual information for the user: 
`f"Insufficient information to decide whether expressions are approximately equal [{val}] vs [{other}]"`

Reply:
Done.